duck/duckdb.go

```diff
@@ -41,20 +41,25 @@ func NewInMemoryDB(opts ...Opts) DuckDB {
 
 // NewDuckDB creates a new DuckDB
 func NewDuckDB(name string, opts ...Opts) DuckDB {
+	exepath := which.Which("duckdb")
+	if exepath == "" {
+		exepath = "/usr/local/bin/duckdb"
+	}
+
 	if len(opts) > 0 {
 		return DuckDB{
 			Name:   name,
 			Mode:   defaultString(opts[0].Mode, "json"),
 			Format: defaultString(opts[0].Format, "parquet"),
-			Path:   defaultString(opts[0].Path, "/usr/local/bin/"),
+			Path:   defaultString(opts[0].Path, exepath),
 			Chunk:  defaultInt(opts[0].Chunk, 0),
 		}
 	}
 	return DuckDB{
 		Name:   name,
 		Mode:   "json",
 		Format: "parquet",
-		Path:   "/usr/local/bin/",
+		Path:   exepath,
```

Comment:
```suggestion
		Exe:   exepath,
```
maybe we should change the name so it is clearly the executable, not the folder it lives in?